### PR TITLE
LLM: Add integration test covering LLM flow

### PR DESCRIPTION
**Pull request by pradh**
Also, return debug info from nodejs server.

### CODE REVIEW

File: server/app_env/integration_test.py [hunk 1]
```diff
@@ -19,4 +19,5 @@ class Config(_base.Config):
   INTEGRATION = True
   API_ROOT = 'https://autopush.api.datacommons.org'
   SECRET_PROJECT = 'datcom-website-dev'
-  SCHEME = 'http'
\ No newline at end of file
+  SCHEME = 'http'
```
Comment (jehangiramjad) [inline server/app_env/integration_test.py:22]: Leaving a random comment for testing

**Review by jehangiramjad — COMMENTED** [review]
Leaving a random comment

**Review by pradh — COMMENTED** [review]
I have reverted the LLM fallback changes - based on Guha discussion, we want to do it slightly differently (follow on PR).

Now this PR is just a small integration test addition.

**Review by jehangiramjad — APPROVED** [review]

**Comment by pradh** [comment]
Thanks!

**Pull request merged**